Elysium/.gitignore

```diff
@@ -1,3 +1,4 @@
+HELP.md
```

Comment:
What is HELP.md?

Reply:
There are no such file in directory, but it surely shouldn't be in gitignore